tools/clang/test/CodeGenDXIL/reflection/type_sys_dump_mat_as_array_of_array.ll

```diff
@@ -0,0 +1,114 @@
+; RUN: %dxa %s -o %t
+; RUN: %dxa %t -dumpreflection | FileCheck %s
+
+target datalayout = "e-m:e-p:32:32-i1:32-i8:32-i16:32-i32:32-i64:64-f16:32-f32:32-f64:64-n8:16:32:64"
+target triple = "dxil-ms-dx"
+
+;struct S2 {
+;    row_major int4x3 m;
+;	int1x3 m1[2];
+;	int3x1 m2;
+;};
+
+; Make sure cbuffer reflection is correct with Vector as array: [rows x [cols x float]] and Vector as array, one row: [cols x float]
```

Comment:
The comment `one row: [cols x float]` seems to refer to the `[3 x i32]`, which is one column, not one row, right?